cachi2/core/package_managers/metayarn.py

```diff
@@ -0,0 +1,25 @@
+from cachi2.core.config import get_config
+from cachi2.core.models.input import Request
+from cachi2.core.models.output import RequestOutput
+from cachi2.core.package_managers import yarn, yarn_classic
+from cachi2.core.package_managers.yarn_classic.main import MissingLockfile, NotV1Lockfile
+from cachi2.core.utils import merge_outputs
+
+
+def fetch_yarn_source(request: Request) -> RequestOutput:
```

Comment:
We need a unit test for this function, as the coverage is very low on this module.